.github/workflows/central_code_quality_check.yml

```diff
@@ -8,14 +8,21 @@ on:
     branches:
       - master
       - main
+      - 4.3
   pull_request:
     branches:
       - master
       - main
+      - 4.3
   workflow_dispatch:
+
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        fetch-depth: [0]
+        java-version: [11]
```

Comment:
using a matrix means there will be more then one option provided. Here, that is not true.  Both are static and we won't be changing that.

Reply:
Thanks, done reverting.